### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by tanjiehui**
Fixes typo and includes the note on Windows 2008 support in the Agent v6 and v7 tab as well.

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/basic_agent_usage/_index.md [hunk 1]
```diff
@@ -150,7 +150,9 @@ When the Agent is running, use the `datadog-agent launch-gui` command to open th
 | [Windows 64-bit][10]                            | Windows 7+                                        |
 | [Windows Azure Stack HCI OS][10]                | All Versions                                      |
 
-**Note**: [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
+**Note**: 
+- [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
+- Windows Server 2008 R2+ is supported, but must have the most recent updates installed in order to run versions 7+ of the Datadog Agent. There is also [a known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```
Comment (hush-hush) [inline content/en/agent/basic_agent_usage/_index.md:155]: I think we would need the update to run the latest version of 6 too, no ?

File: content/en/agent/basic_agent_usage/_index.md [hunk 2]
```diff
@@ -150,7 +150,9 @@ When the Agent is running, use the `datadog-agent launch-gui` command to open th
 | [Windows 64-bit][10]                            | Windows 7+                                        |
 | [Windows Azure Stack HCI OS][10]                | All Versions                                      |
 
-**Note**: [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
+**Note**:
```
Comment (ruthnaebeck) [inline content/en/agent/basic_agent_usage/_index.md:153]:
```suggestion
**Notes**: 
```

File: content/en/agent/basic_agent_usage/_index.md (hunk 1 above)
Comment (ruthnaebeck) [inline content/en/agent/basic_agent_usage/_index.md:155]:
```suggestion
- Windows Server 2008 R2+ is only supported on the most recent Datadog Agents v6 and v7. However, there is a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```

File: content/en/agent/basic_agent_usage/_index.md [hunk 3]
```diff
@@ -184,7 +186,7 @@ When the Agent is running, use the `datadog-agent launch-gui` command to open th
 
 - [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
 
-- Windows Server 2008 R2+ is supported, but must have the most recent updates installed in order to run versions 7+ of the Datadog Agent. There is also [a known issued with clock drift and Go][12] that affects Windows Server 2008 R2+.
+- Windows Server 2008 R2+ is supported, but there is [a known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```
Comment (ruthnaebeck) [inline content/en/agent/basic_agent_usage/_index.md:189]:
```suggestion
- Windows Server 2008 R2+ is supported, but there is a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

File: content/en/agent/basic_agent_usage/_index.md (hunk 1 above)
Comment (tanjiehui) [reply content/en/agent/basic_agent_usage/_index.md:155]: The issue referenced is specific to Python3, some Windows Updates have to be installed for it to work. Agent 6 is fine as long as they are using Python2.7.

Comment (tanjiehui) [reply content/en/agent/basic_agent_usage/_index.md:155]: @ruthnaebeck I want to clarify that by "must have the most recent updates installed", it's referring to Windows update and not Agent update. How would you suggest we make this clear?

Comment (ruthnaebeck) [reply content/en/agent/basic_agent_usage/_index.md:155]: @tanjiehui - Updated my feedback

File: content/en/agent/basic_agent_usage/_index.md [hunk 4]
```diff
@@ -150,7 +150,9 @@ When the Agent is running, use the `datadog-agent launch-gui` command to open th
 | [Windows 64-bit][10]                            | Windows 7+                                        |
 | [Windows Azure Stack HCI OS][10]                | All Versions                                      |
 
-**Note**: [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
+**Notes**: 
+- [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
+- Datadog Agent v7+ supports Windows Server 2008 R2+ with the most recent Windows updates installed. There is also a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```
Comment (olivielpeau) [inline content/en/agent/basic_agent_usage/_index.md:155]: Unless I'm missing something this note only applies to Windows Server 2008 R2 specifically, not later Windows versions:

```suggestion
- Datadog Agent v7+ supports Windows Server 2008 R2 with the most recent Windows updates installed. There is also a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2.
```

File: content/en/agent/basic_agent_usage/_index.md [hunk 5]
```diff
@@ -184,7 +186,7 @@ When the Agent is running, use the `datadog-agent launch-gui` command to open th
 
 - [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
 
-- Windows Server 2008 R2+ is supported, but must have the most recent updates installed in order to run versions 7+ of the Datadog Agent. There is also [a known issued with clock drift and Go][12] that affects Windows Server 2008 R2+.
+- Windows Server 2008 R2+ is supported, but there is a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```
Comment (olivielpeau) [inline content/en/agent/basic_agent_usage/_index.md:189]: Ditto:

```suggestion
- Windows Server 2008 R2 is supported, but there is a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2.
```

**Review by ruthnaebeck — APPROVED** [review]

**Pull request merged**